### DISCUSSION TITLE
Joomla user groups

### DISCUSSION THREAD

**Discussion by CJ-Hurc** [nette/tracy#588] [Ideas]
It would be useful for your joomla integration to allow user groups or users list in addition to the user ip address. Being that in our situation, our ip address other users which might not be developers would still get debugging and dumps. I know you can change access levels on the plugin, but that could break things if we call a dump function when developing, and then the acl doesn't load the class due to acl, the site would break. So i think adding those features would be beneficial outside of using the debugger for general error page information.  I assume that through using the extension we can still use it the same way we could if installed through composer and call the dump methods directly in our code.

Thoughts? 

Thanks!

**Comment by mabar** [comment]
You should ask for that in the Joomla integration repository, not here.
Tracy itself does not limit how you enable the debug mode.

**Comment by CJ-Hurc** [comment]
Thanks! Didn't realize that wasn't you.

⁣Get BlueMail for Android ​
